baselines/ppo2/ppo2.py

```diff
@@ -364,31 +378,37 @@ def fn(x):
             mdp_gen_params = additional_params["mdp_generation_params"]
             mdp_fn = LayoutGenerator.mdp_gen_fn_from_dict(mdp_params=mdp_params, **mdp_gen_params)
             overcooked_env = OvercookedEnv(mdp=mdp_fn, **env_params)
-            agent = get_agent_from_model(model, additional_params["sim_threads"], is_joint_action=(run_type == "joint_ppo"))
+            agent = get_agent_from_model(model, additional_params["sim_threads"], is_joint_action=(run_type == "joint_ppo"), return_action_probs=False)
             agent.set_mdp(overcooked_env.mdp)
 
-            if run_type == "ppo":
-                if additional_params["OTHER_AGENT_TYPE"] == 'sp':
-                    agent_pair = AgentPair(agent, agent, allow_duplicate_agents=True)
+            if not additional_params["OTHER_AGENT_TYPE"] == 'tom':  # For TOM we vizualise
+                # and also evaluate the performance of the ppo with various agents in section "if update % log_interval"
```

Comment:
Is there a reason for this? It would be nicer to visualize all agents in the same location of the code

Reply:
Oh I see, it's because of the way logging is implemented. I guess this is fine, it's just a lot of nested if statements 😢

Reply:
Yep I agree that it's ugly -- apologies for that. I wanted to ensure that my changes wouldn't break ppo2.py, and adding this extra nested statement was the best solution I could think of :(